src/main/java/frc/robot/commands/ToggleRotationTarget.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.DrivetrainSubsystem;
+
+public class ToggleRotationTarget extends CommandBase{
+    
+    private DrivetrainSubsystem drivetrainSubsystem;
+    private Rotation2d target;
+
+    public ToggleRotationTarget(DrivetrainSubsystem drivetrainSubsystem, Rotation2d target) {
```

Comment:
I think we need to capture the second argument as a supplied rotation target instead of a constant arguement